jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -830,6 +830,26 @@ def set_center(self, new_cen, subset_name=None, update=False):
             self.subset_definitions = []
             self.subset_definitions = tmp
 
+    def set_center(self, new_cen, subset_name=None):
```

Comment:
Hmm I thought we agreed to just remove it from the public API exposure?

Reply:
that's effectively what this does, but we want it to always be True now, whereas the internal method defaults to False

Reply:
I was thinking more along the line of a one-line change here:

https://github.com/spacetelescope/jdaviz/blob/e6548c4bf6101cc23228665ee2317692a8f0ab32/jdaviz/configs/default/plugins/subset_tools/subset_tools.py#L177

Reply:
oh that removes `set_center` entirely, we don't want to do that, we just don't want to expose the `update` keyword argument

Reply:
did we decide that we didn't want it exposed at all anymore?  Maybe I just misunderstood/misremembered

Reply:
I thought we said it shouldn't be public API in the first place, right, @javerbukh ?

Reply:
Yes it shouldn't be public since it doesn't really make sense as API. The Recenter button provides a new center (as an estimate) which is what the subset is then centered on. Using that method with API would require the user to come up with the center themselves and at that point they should just use the update subset public API (which does not exist yet).

Reply:
ok, then I can revert that here and maybe we should wait until the alternative (upcoming) edit subset API is in place and then deprecate this method then?